lib/matplotlib/tests/test_simplification.py

```diff
@@ -173,7 +171,7 @@ def test_throw_rendering_complexity_exceeded():
     try:
         fig.savefig(io.BytesIO())
     finally:
-        rcParams['path.simplify'] = True
+        plt.rcParams['path.simplify'] = True
```

Comment:
I don't know; I just backported the warning fixes from master (i.e., removal of `pylab`.)